hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestHoodieOptionConfig.scala

```diff
@@ -109,16 +109,6 @@ class TestHoodieOptionConfig extends SparkClientFunctionalTestHarness {
         StructField("dt", StringType, true))
     )
 
-    // miss primaryKey parameter
-    val sqlOptions1 = baseSqlOptions ++ Map(
-      "type" -> "mor"
-    )
-
-    val e1 = intercept[IllegalArgumentException] {
-      HoodieOptionConfig.validateTable(spark, schema, sqlOptions1)
```

Comment:
add it back to make the test pass.

Reply:
we can't add this back. after this patch, record key field is not a mandatory field in general.